externs/browser/w3c_dom1.js

```diff
@@ -473,6 +473,24 @@ NodeList.prototype.item = function(index) {};
  */
 NodeList.prototype.forEach = function(callback, opt_thisobj) {};
 
+/**
+ * @return {!Iterator<!Array<number|T>>}
```

Comment:
BTW, these should be `IteratorIterable` instead of just `Iterator`.
I've fixed that in the version I've imported, so no need to fix it yourself.

Reply:
Oh, good catch